src/accessory.ts

```diff
@@ -165,6 +165,7 @@ export class DaikinCloudAirConditioningAccessory {
         this.platform.log.info(`[${this.name}] SET TargetHeaterCoolerState, daikinOperationMode to: ${daikinOperationMode}`);
         await this.accessory.context.device.setData('climateControl', 'operationMode', daikinOperationMode);
         await this.accessory.context.device.updateData();
+        await this.handleActiveStateSet('on')
```

Comment:
I would rather not call the handlers from within another handler.
Can you change the code so the AC is turned on like in the `handleActiveStateSet` handler?
And move this one line up, `updateData` should be at the bottom.

So:
````
await this.accessory.context.device.setData('climateControl', 'onOffMode', 'on');
await this.accessory.context.device.updateData();
```

Reply:
And sorry for the late reply, holiday with the family :)

Reply:
Hi @JeroenVdb ,
Changes done!